yarn.lock

```diff
@@ -341,6 +341,13 @@
     stream-browserify "^3.0.0"
     util "^0.12.4"
 
+"@cspotcode/source-map-support@^0.8.0":
```

Comment:
just a reminder to dedupe if you haven't - i think @zzmp set up something nice in interface but not sure if we have it here

Reply:
Yeah, I don't think we run deduplication in this project yet (judging by the absence of this tool). I would be more than happy to do it, but I think I would do it in a separate PR (can be a follow-up) to make it a bit easier to review! What do you think?

Reply:
Another one high on my list is:
- adding ESLint Uniswap config
- fixing local development with Lerna (local development is slow right now, lots of false positive Typescript errors)

Reply:
yup agree it's a separate concern from this PR

Reply:
I am going to follow-up with this and ESLint because it would make my life easier. And I am blocked on some WC responses anyway.

Reply:
For now, I run it with `npx` just on this PR specifically. Also, note that most of the lock file changes are only affecting our root/workspace package, since we take a heavy advantage of peer dependencies. This will help understand the lock file a bit more, but it's not something that is going to have a direct impact on the clients - they will have to deduplicate on their own.

Reply:
Going forward, I will do `yarn deduplicate` as a lerna command on every sub-package.